.gitignore

```diff
@@ -82,3 +82,7 @@ report.xml
 tsconfig.tsbuildinfo
 
 docs/
+
+# Proto
+.protoc/
+.swift-protobuf/
```

Comment:
Please add a new line at the end in here, `.editorconfig` and `scripts/compile_protos.mk` to get rid of the warning